content/docs/command-reference/remove.md

```diff
@@ -13,35 +14,32 @@ positional arguments:
 
 ## Description
 
-This command safely removes stages from
-[dvc.yaml](/doc/user-guide/dvc-files-and-directories#dvcyaml-file), their
-`.gitignore` entries, and optionally removes from the <abbr>workspace</abbr>
-files or directories that are tracked by DVC. It takes one or more stage names
-(see `-n` option of `dvc run`) or `.dvc` files as target, removes it, and
-optionally removes all of its outputs (`outs` field).
+This command can safely remove stages from `dvc.yaml` or `.dvc` files. This
+includes deleting the appropriate `.gitignore` entries, and optionally the
+actual output files they track (see `--outs` option).
 
-If there are no stages left in
-[dvc.yaml](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) after stage
-removal then both
-[dvc.yaml](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) and
-[dvc.lock](/doc/user-guide/dvc-files-and-directories#dvclock-file) are removed
-from the workspace.
+It takes one or more stage names (see `-n` option of `dvc run`) or `.dvc` file
+names as `targets`.
 
-Note that in the case when `targets` are `.dvc` files, the tracked files or
-directories (`outs` in the `.dvc` file) are _not_ removed by this command unless
-`--outs` is specified.
+If there are no stages left in `dvc.yaml` after the removal, then both
+`dvc.yaml` and `dvc.lock` are deleted.
 
-`dvc remove` does not remove files from the DVC cache or remote storage (see
-`dvc gc`). However, remember to run `dvc push` to save the files you actually
-want to use or share in the future.
+Note that the actual <abbr>output</abbr> files or directories tracked by the
+stage (`outs` field) are not removed by this command, unless the `--outs` option
+is used.
 
-Refer to [Updating Tracked Files](/doc/user-guide/how-to/update-tracked-files)
-to see how it can be used to replace or modify files that are tracked by DVC.
+> `dvc remove` doesn't remove files from the DVC <abbr>cache</abbr> or
```

Comment:
should be at the very top I think - very important for people to understand that the can always revert changes in Git and do `dvc checkout` to restore their data

Reply:
if data is cached of course

Reply:
OK I'm moving the note near the top, but I didn't understand the relationship between the text in the note and what you're describing... Let me 🤔

Reply:
> can always revert changes in Git and do dvc checkout to restore their data

Yeah sorry I'm not seeing the connection to `remove` here. But the note is up now.